### PR TITLE
Added Freepic icons resource.

### LINKED ISSUE #1993: [ADD] Freepic Icons  recourse link
### Site URL

https://www.freepik.com/icons

### Description of the above provided site

This provides a 10M+ free PNG and SVG icons, including the one you have in mind.

### Category

Frontend

### Sub Category

icons

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by Hemant-Lohar**
closes #1993 

---

Updated json file for frontend/icons category.

image

Review and merge @Anmol-Baranwal @CBID2

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Looks good to me

**Comment by CBID2** [comment]
Did you raise this as an issue @Hemant-Lohar?

**Comment by Hemant-Lohar** [comment]
@CBID2 It's my bad I didn't follow all the guidelines provided in the contributions guidelines. I will correct it. Should I create a new pull request?

**Comment by CBID2** [comment]
> @CBID2 It's my bad I didn't follow all the guidelines provided in the contributions guidelines. I will correct it. Should I create a new pull request?
> 
> 

Not yet, just raise the issue and I'll review it. If it gets approved, I'll reopen your PR. If not, you can create a new one.

**Comment by Hemant-Lohar** [comment]
I had raised the issue it was #1993

**Comment by CBID2** [comment]
> I had raised the issue it was #1993 @CBID2

Great @Hemant-Lohar. Now just add it to the PR form. I highly recommend reading [Writing Your First Pull Request: Tips, Best Practices, and AI-Powered Tools for Success](https://dev.to/opensauced/writing-your-first-pull-request-tips-best-practices-and-ai-powered-tools-for-success-3bg9) to help you fill it out.

File: database/frontend/icons.json [hunk 1]
```diff
@@ -237,11 +237,16 @@
     "category": "frontend",
     "subcategory": "icons"
   },
+  {
+    "name": "Freepik Icons",
+    "description": "The biggest free icon library in the world. Find icons that go together Fast. It has 10M+ ready-to-use, free license icons.",
+    "url": "https://www.freepik.com/icons",
```
Comment (CBID2) [inline database/frontend/icons.json:243]: [proofreading] Don't forget to add the `category` and `subcategory` names. Refer to [this section of our Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md#appending-new-links-) for more context. 
```suggestion
    "name": "Freepik Icons",
    "description": "The biggest free icon library in the world. Find icons that go together Fast. It has 10M+ ready-to-use, free license icons.",
    "url": "https://www.freepik.com/icons",
    "category": "frontend",
    "subcategory: "icons"
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Hemant-Lohar. I made some suggestions to improve your PR. Other than that you're off to a great start! :)

**Comment by Anmol-Baranwal** [comment]
@Hemant-Lohar 
Following the PR template is a must. It includes special words like 'fixes' or 'resolves' that connect your issue to the PR, as explained in the official docs.

If you don't link the issue correctly, even after the PR is merged, the issue might stay open, causing huge problems.

I'm doing it for you this time. Please ensure it next time.

Just a heads-up to help you link issue properly next time. Have a great day!

**Comment by Anmol-Baranwal** [comment]
@Hemant-Lohar 
Are you interested in contributing?
If not, please let me know so that I can close the issue.

**Pull request closed without merging**